priv/repo/migrations/20160207012812_copy_hospital_csv.exs

```diff
@@ -33,10 +34,28 @@ defmodule HospitalReadmissionServer.Repo.Migrations.CopyHospitalCsv do
       WITH CSV HEADER;
     """
 
+    # Set integers with value of -1 to NULL
     execute """
       UPDATE hospitals
-        SET inserted_at = current_time,
-        updated_at = current_time
+        SET number_of_discharges = NULL
+        WHERE number_of_discharges = -1;
+    """
+    execute """
+      UPDATE hospitals
+        SET provider_number = NULL
+        WHERE provider_number = -1;
+    """
+    execute """
+      UPDATE hospitals
+        SET number_of_readmissions = NULL
+        WHERE number_of_readmissions = -1;
+    """
+
+    # elixir writes timestamps without timezone  = postgres.localtimestamp
+    execute """
+      UPDATE hospitals
+        SET inserted_at = localtimestamp,
+        updated_at = localtimestamp
```

Comment:
Is that really why? Or is it just because we're not explicitly setting them befor now?